examples/native-fungible/src/lib.rs

```diff
@@ -1,4 +1,50 @@
 // Copyright (c) Zefchain Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+/*!
+# Native Fungible Token Example Application
+
+This app is very sumilar to the [Fungible Token Example Application](https://github.com/linera-io/linera-protocol/blob/main/examples/fungible/README.md#fungible-token-example-application). The difference is that this is a native token that will use system API calls for operations.
+The general aspects of how it works can be referred to the linked README.
+
+# How It Works
+
+Refer to [Fungible Token Example Application - How It Works](https://github.com/linera-io/linera-protocol/blob/main/examples/fungible/README.md#how-it-works).
+
+# Usage
+
+## Setting Up
+
+Most of this can also be referred to the [fungible app README](https://github.com/linera-io/linera-protocol/blob/main/examples/fungible/README.md#setting-up), except for at the end when compiling and publishing the bytecode, what you'll need to do will be slightly different:
+
+Now, compile the `native-fungible` application WebAssembly binaries, and publish them as an application
+bytecode:
+
+```bash
+(cd examples/native-fungible && cargo build --release)
+
+BYTECODE_ID=$(linera publish-bytecode \
+    examples/target/wasm32-unknown-unknown/release/native_fungible_{contract,service}.wasm)
+```
+
+Here, we stored the new bytecode ID in a variable `BYTECODE_ID` to be reused it later.
+
+## Creating a Token
+
+Most of this can also be referred to the [fungible app README](https://github.com/linera-io/linera-protocol/blob/main/examples/fungible/README.md#creating-a-token), except for at the end when creating the application, you don't need to pass a `ticker_symbol` because for the Native Fungible App that will be hardcoded to `NAT`.
+
+```bash
+APP_ID=$(linera create-application $BYTECODE_ID \
+    --json-argument "{ \"accounts\": {
+        \"User:$OWNER_1\": \"100.\"
+    } }" \
+    --json-parameters "{ \"ticker_symbol\": \"NAT\" }" \
+)
```

Comment:
Nit: same here :sweat_smile: 

```suggestion
APP_ID="$(linera create-application $BYTECODE_ID \
    --json-argument "{ \"accounts\": {
        \"User:$OWNER_1\": \"100.\"
    } }" \
    --json-parameters "{ \"ticker_symbol\": \"NAT\" }" \
)"
```

Reply:
Doing double quotes here is tricky because it'll remove the simplification of doing single quotes for the option arguments, because we would have to escape all the double quotes again